src/modules/features/SettingDialogue/settingConstants.ts

```diff
@@ -105,48 +105,50 @@ export const SettingInformation: Partial<Record<keyof AllSettings, Configuration
     },
     syncOnSave: {
         name: "Sync on Save",
-        desc: "When you save a file, sync automatically",
+        desc: "Forces a sync of the file when it's saved.",
```

Comment:
Sorry for the ambiguity, but I'm glad your PR uncovered it. This simply starts synchronisation when we save the file, so the file that is synchronised is not limited to this file.

Reply:
changed to `Starts synchronisation when a file is saved.`